README.md

```diff
@@ -257,25 +178,13 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+Micro-Verse students 
```

Comment:
- [ ] [OPTIONAL] Good job adding a link to the MIT Licence into your README, but please make sure to remove _`Note:`_ from the LICENCE section of the README. Note is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it

<img width="975" alt="Screenshot 2023-03-21 at 9 54 02 PM" src="https://user-images.githubusercontent.com/44798044/226684107-29073634-34b7-447e-8c46-3983298cf22b.png">